__tests__/src/utils/draftutil.test.js

```diff
@@ -72,31 +72,31 @@ describe('getDraftFormat', () => {
         }
         expectedFormat.push([filterText]);
       });
-      return expectedFormat;
+      return { sealed: false, trash: 0, filters: expectedFormat };
```

Comment:
no `pickAtTime` or `cards` in any tests?

Reply:
cards is only for compiled formats not the descriptions for them. Was missing the pickAtTimes though.